README.md

```diff
@@ -48,6 +48,21 @@ create-scxa-analytics-schema.sh
 
 You can override the default target Solr collection name by setting `SOLR_COLLECTION`, but remember to include the additional `v<schema-version-number>` at the end, or the loader might refuse to load this.
 
+## Add suggesters
```

Comment:
Is it needed to add the suggesters right after creating the schema or this should go after experiments data has been loaded? Or it doesn't matter? If it should happen after experiments are created, then I suggest that the docs for this moves after the part where we load experiments.